### ISSUE TITLE
[BUG] Apache config for mail sites won't allow the use of .htaccess files in Rainloop.

### ISSUE THREAD

**Issue by plutocrat** [hestiacp/hestiacp#1915]
**Describe the bug**
Apache config for mail sites won't allow the use of .htaccess files in Rainloop. 
Contains a stanza to enable it for Roundcube. Needs similar for Rainloop. 

**To Reproduce**
Creating /var/lib/rainlooop/.htaccess with any content causes immediate apache 500 error. 
Altering the config for mail domain in /home/user/conf/mail/domain/apache2.conf and apache2.ssl.conf allows its use. 
eg. 
```
<Directory /var/lib/roundcube/>
    Options +FollowSymLinks
    # This is needed to parse /var/lib/roundcube/.htaccess. See its
    # content before setting AllowOverride to None.
    AllowOverride All
    order allow,deny
    allow from all
</Directory>

# Protecting basic directories:
<Directory /var/lib/rainloop/data>
        Options -FollowSymLinks
        AllowOverride None
</Directory>
```
Needs to be 
```
<Directory /var/lib/roundcube/>
    Options +FollowSymLinks
    # This is needed to parse /var/lib/roundcube/.htaccess. See its
    # content before setting AllowOverride to None.
    AllowOverride All
    order allow,deny
    allow from all
</Directory>

# Allow rainloop htaccess
<Directory /var/lib/rainloop/>
        Options +FollowSymLinks
        AllowOverride All
</Directory>

# Protect rainloop data directories:
<Directory /var/lib/rainloop/data>
        Options -FollowSymLinks
        AllowOverride None
</Directory>
```

**Hestia Control Panel version:**
Latest

**Additional context**
Debugging this wasn't particularly easy, as no Apache logs are created. Nginx logs for mail domains are available, but they don't tell the whole story. You might consider adding logs to the Apache config under the SSL section too i.e. 
CustomLog "/var/log/apache2/domains/mail.domain.com.log" combined
ErrorLog "/var/log/apache2/domains/mail.domain.com.error.log"
But as regular websites are under /var/log/apache2/domains and mail websites are under /var/log/nginx/domains/ there's no clear place to put them. Just a suggestion anyway.

**Comment by jaapmarcus** [comment]
I see an issue in https://github.com/hestiacp/hestiacp/blob/0d49ebc3438fb7ff34d329bd72c7467f7b878c57/install/deb/templates/mail/apache2/rainloop.stpl#L13-L20

https://github.com/hestiacp/hestiacp/blob/main/install/deb/templates/mail/apache2/rainloop.tpl

Should be fine

**Comment by jaapmarcus** [comment]
https://github.com/hestiacp/hestiacp/pull/1916/commits/6e9af86acaee2126c2c84c6e54b27499d95b053f

Will be released with 1.4.3
